### PR TITLE
Header: Communities incorporation

### PR DESCRIPTION
**Pull request by NRodriguezcuellar**
Incorporated menu items for the communities feature

mobile =>
Screenshot from 2022-01-21 16-50-32
tablet =>
Screenshot from 2022-01-21 16-34-59
desktop =>
Screenshot from 2022-01-21 16-34-47
Screenshot from 2022-01-21 16-54-55

closes https://github.com/inveniosoftware/invenio-app-rdm/issues/1172

### CODE REVIEW

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header_login.html [hunk 1]
```diff
@@ -6,68 +6,70 @@
...
-      <a href=" {{ url_for_security('logout') }}" class="ui button"><i
-              class="sign-out icon"></i> {{ _('Log out') }}</a>
+        {%- else %}
+            <a href=" {{ url_for_security('logout') }}" class="ui button"><i
+                    class="sign-out icon"></i> {{ _('Log out') }}</a>
+        {%- endif %}
     {%- endif %}
-  {%- endif %}
-{%- endif %}
+{%- endif %}
```
Comment (Pineirin) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header_login.html:75]: I think you should add a new line at the end of the file.

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header_frontpage.html [hunk 2]
```diff
@@ -12,6 +12,7 @@
 {#- Remove search field from top bar on frontpage #}
 {%- block navbar_search %}{% endblock %}
 
+
```
Comment (Pineirin) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header_frontpage.html:15]: Maybe revert this change to keep the commit shorter?

**Review by Pineirin — COMMENTED** [review]
In general it looks good to me (a bit hard to follow the changes inside all those ifs).
Could you maybe reduce the space between communities and dashboard in the mobile view.

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header_login.html (hunk 1 above)
Comment (NRodriguezcuellar) [reply invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header_login.html:75]: will do :+1:

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header_frontpage.html (hunk 2 above)
Comment (NRodriguezcuellar) [reply invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header_frontpage.html:15]: i agree!

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less [hunk 3]
```diff
@@ -94,6 +101,19 @@ nav {
   display: block;
 }
 
+.active .navbar-item.navbar-button.navbar-actions-container {
```
Comment (ntarocco) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less:104]: What happens if a new submenu is created dynamically via `flask-menu`?

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header.html [hunk 4]
```diff
@@ -46,47 +49,63 @@
...
-        </ul>
-      </div>
-      </nav>
+              {%- endblock navbar_nav %}
+              <li class="navbar-item navbar-button navbar-actions-container">
+                <ul class="rdm-sign-buttons">
+                  {%- block navbar_right %}
+                    {%- if config.ACCOUNTS and current_user.is_authenticated %}
+                      <li class="navbar-item dashboard-button ">
+                        <a class="navbar-button " href="/dashboard">My dashboard</a>
```
Comment (ntarocco) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header.html:79]: I think that this should be injected from the backend and not hardcoded.
We are also not taking into account translations here, which is a must have when adding labels.

File: invenio_app_rdm/theme/views.py [hunk 5]
```diff
@@ -35,11 +35,7 @@ def create_blueprint(app):
     @blueprint.before_app_first_request
     def init_menu():
         """Initialize menu before first request."""
-        item = current_menu.submenu("main.deposit")
-        item.register(
-            "invenio_app_rdm_records.deposit_search", _("Uploads"), order=1)
-
-        item = current_menu.submenu('plus.deposit').register(
+        current_menu.submenu('plus.deposit').register(
```
Comment (ntarocco) [inline invenio_app_rdm/theme/views.py:38]: are we sure that we want to remove this from here and hardcode in in the template?

ping @zzacharo to have another opinion

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less (hunk 3 above)
Comment (NRodriguezcuellar) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less:104]: It should not interfere with the existing or new submenu's styling as everything scoped

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header.html (hunk 4 above)
Comment (NRodriguezcuellar) [reply invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header.html:79]: that makes sense

File: invenio_app_rdm/theme/views.py (hunk 5 above)
Comment (NRodriguezcuellar) [reply invenio_app_rdm/theme/views.py:38]: What i deleted here is just the upload button that was on the left main nav portion, as was suggested in the UX. But i agree as discussed earlier that i can make a new submenu to make it dynamic
